app.js

```diff
@@ -16,7 +16,18 @@ function Projects (title, client, author, published, description, tech, image){
 
 
 
-// next i'll need...maybe not a prototype, but some function that pushes items through the constructor so that my slideshow can populate.
+// next i'll need a prototype.. some function that pushes items through the constructor so that my slideshow can populate.
```

Comment:
Focus on the basic functionality first, just get the information rendering to the page. It'll look really similar to the blog for now, and then we can add the slide show later on. As long as you have the rendering done, the slide show will basically overlay that when you're ready to implement.